.github/workflows/cypress-workflow.yml

```
@@ -65,7 +65,7 @@ jobs:
           config: baseUrl=${{ inputs.test-endpoint }}
           env: true
         env:
-          CYPRESS_STATE_USER_3: ${{ secrets.cypress-user3 }}
+          CYPRESS_STATE_USER_3: ${{ secrets.cypress-user2 }}
```

you'll need to change the left-hand side to number 2 here as well since you updated the cypress commands to use state user 2
```suggestion
          CYPRESS_STATE_USER_2: ${{ secrets.cypress-user2 }}
```

---

thanks for catching this! fixed!